app.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "Reaction Commerce",
+  "description": "Reaction is a modern reactive, real-time event driven ecommerce platform.",
+  "repository": "https://github.com/reactioncommerce/reaction",
+  "logo": "https://avatars1.githubusercontent.com/u/5605462?s=200&v=4",
+  "website": "https://www.reactioncommerce.com",
+  "keywords": ["reaction", "e-commerce", "meteor"],
+  "buildpacks": [
+    {
+      "url": "https://github.com/Zanobo/reaction-buildpack.git"
```

Comment:
I think we want to start using our fork of this even though it doesn't contain any changes. Just to keep it stable.

Reply:
We already have a fork